src/derivative_operators/abstract_operator_functions.jl

```diff
@@ -209,3 +209,16 @@ function Base.sparse(A::AbstractDerivativeOperator{T}) where T
     end
     return mat
 end
+
+#=
+    Fallback methods that use the full representation of the operator
+
+    As with the convention for regular matrices, right division is defined in 
+    terms of left division. (The result may not be correct for some BC as the 
+    transpose of a derivative operator is currently defined to be a no-op)
+=#
+Base.expm(A::AbstractDerivativeOperator{T}) where T = expm(full(A))
+Base.:\(A::AbstractVecOrMat, B::AbstractDerivativeOperator) = A \ full(B)
+Base.:\(A::AbstractDerivativeOperator, B::AbstractVecOrMat) = full(A) \ B
+Base.:/(A::AbstractVecOrMat, B::AbstractDerivativeOperator) = (B' \ A')'
```

Comment:
These transposes might be incorrect because they are just implemented as a no-op. Sorry if I was unclear before. I think you should just `full` and `/`, or does the generic version but `full` before transpose.